src/phpDocumentor/Reflection/Php/Factory/Property.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @copyright 2010-2015 Mike van Riel<mike@phpdoc.org>
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor\Reflection\Php\Factory;
+
+use InvalidArgumentException;
+use phpDocumentor\Descriptor\Property as PropertyDescriptor;
+use phpDocumentor\Reflection\Fqsen;
+use phpDocumentor\Reflection\Php\Factory;
+use phpDocumentor\Reflection\Php\ProjectFactoryStrategy;
+use phpDocumentor\Reflection\Php\StrategyContainer;
+use phpDocumentor\Reflection\Php\Visibility;
+use PhpParser\Node\Stmt\Property as PropertyNode;
+
+/**
+ * Strategy to convert Param to Argument
+ *
+ * @see PropertyDescriptor
+ * @see PropertyNode
+ */
+final class Property implements ProjectFactoryStrategy
+{
+
+    /**
+     * Returns true when the strategy is able to handle the object.
+     *
+     * @param object $object object to check.
+     * @return boolean
+     */
+    public function matches($object)
+    {
+        return $object instanceof PropertyNode;
+    }
+
+    /**
+     * Creates an PropertyDescriptor out of the given object.
+     * Since an object might contain other objects that need to be converted the $factory is passed so it can be
+     * used to create nested Elements.
+     *
+     * @param PropertyNode $object object to convert to an PropertyDescriptor
+     * @param StrategyContainer $strategies used to convert nested objects.
+     * @return PropertyDescriptor
+     *
+     * @throws InvalidArgumentException when this strategy is not able to handle $object
+     */
+    public function create($object, StrategyContainer $strategies)
+    {
+        if (!$this->matches($object)) {
+            throw new InvalidArgumentException(
+                sprintf('%s cannot handle objects with the type %s',
+                    __CLASS__,
+                    is_object($object) ? get_class($object) : gettype($object)
+                )
+            );
+        }
+
+        $visibility = $this->buildVisibility($object);
+
+        return new PropertyDescriptor(new Fqsen($object->name), $visibility, null, $object->default, $object->static);
```

Comment:
The name property is only a local name while Fqsen is the complete name including namespace and class; so with the method `myMethod` in the class `MyNamespace\MyClass` will `$object->name` return `myMethod` while what you want is `\MyNamespace\MyClass::myMethod()`